README.md

```diff
@@ -69,22 +71,77 @@ recompile
 
 ```
 
-From the IEX shell `iex(1)> `, etc. 
+_From the IEX shell `iex(1)> `, etc._ 
+
+### Set the variables for the run
+**TODO This is still a bit complicated, could probably deduce the starting/ending indexes...**
 
-### Massaging a single file (for testing)
+Setup the variables that you'll use for the run. For example:
```

Comment:
This is the "easy" example that someone could follow to convert files. Note that we don't require code changes any more.